modules/nf-core/nacho/qc/main.nf

```diff
@@ -0,0 +1,79 @@
+nextflow.enable.moduleBinaries = true
```

Comment:
Better to have this added into the tests/nextflow.config file and then adding the line "config "./nextflow.config" to the main.nf.test file. Just because the docs https://www.nextflow.io/docs/latest/module.html#module-binaries say to add them in the config or pipeline file, and also because no other module seems to set this type of parameters directly in the main.nf file